Makefile

```diff
@@ -125,6 +125,19 @@ test_all:: test_pkg test_integration
 test_lifecycle:
 	@cd pkg && $(GO_TEST) github.com/pulumi/pulumi/pkg/v3/engine/lifecycletest
 
+LIFECYCLE_TEST_FUZZ_CHECKS = 1000
```

Comment:
We seem to mostly declare variables at the top of the file, maybe this should be there as well?

Reply:
Agreed.  I also thought you need to use `?=` instead of `=`, to be able to override this? `make` is surprising me every day.

Reply:
I will move it.

`?=` means assign only if there is no value, which means you can override both with an environment variable (which Make will auto turn into a Make variable), as well as with Make overriding on the command line. I will opt for this then since perhaps we prefer that flexibility?

In terms of what you can override, I believe any variable can be overridden on the Make command line (that is, with `make foo=bar`, not with env vars) unless it is declared with `override`.